frontend/src/component/personalDashboard/PersonalDashboard.tsx

```diff
@@ -175,6 +183,49 @@ export const PersonalDashboard = () => {
                 </ViewKeyConceptsButton>
             </WelcomeSection>
 
+            {showTimelinePanel ? (
+                <SectionAccordion
+                    disableGutters
+                    expanded={expandTimeline ?? true}
+                    onChange={() => toggleSectionState('timeline')}
+                >
+                    <StyledAccordionSummary
+                        expandIcon={
+                            <ExpandMore titleAccess='Toggle timeline section' />
+                        }
+                        id='timeline-panel-header'
+                        aria-controls='timeline-panel-content'
+                    >
+                        <AccordionSummaryText>
+                            <AccordionSummaryHeader>
+                                Timeline of events
+                            </AccordionSummaryHeader>
+                            <AccordionSummarySubtitle>
+                                Overview of recent activities across all
+                                projects in Unleash. Make debugging easier and{' '}
+                                <Link
+                                    to={signalsLink}
+                                    onClick={() => {
+                                        trackEvent('event-timeline', {
+                                            props: {
+                                                eventType: 'signals clicked',
+                                            },
+                                        });
+                                    }}
+                                >
+                                    include external signals
+                                </Link>
+                                {''} to get a fuller overview.
```

Comment:
Is this doing anything? Maybe we want this instead?
```suggestion
                                {' '}to get a fuller overview.
```